### PR TITLE
Initial commit of asset queuing

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

Adds initial support for asset queuing.

### CODE REVIEW

File: bin/cli-jobs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+program
+  .command('process')
+  .description('starts job processing')
+  .action(processJobs);
+
+program.parse(process.argv);
+
+// If there is no command listed, output help.
+if (!process.argv.slice(2).length) {
```
Comment (impronunciable) [inline bin/cli-jobs]: `if (process.argv.length <= 2)`

File: bin/cli-serve [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+ * Setup the debug paramater.
+ */
+
+process.env.DEBUG = process.env.TALK_DEBUG;
+
+const app = require('../app');
+const debug = require('debug')('talk:server');
+const http = require('http');
+const init = require('../init');
+const port = normalizePort(process.env.TALK_PORT || '3000');
```
Comment (impronunciable) [inline bin/cli-serve]: We can move this to line 21

File: bin/cli-serve [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  switch (error.code) {
+  case 'EACCES':
+    console.error(`${bind} requires elevated privileges`);
+    break;
+  case 'EADDRINUSE':
+    console.error(`${bind} is already in use`);
+    break;
+  }
+
+  throw error;
```
Comment (impronunciable) [inline bin/cli-serve:51]: this could be the default case

File: client/coral-embed-stream/src/CommentStream.js [hunk 4]
```diff
@@ -61,8 +61,9 @@ class CommentStream extends Component {
     // Set up messaging between embedded Iframe an parent component
     // Using recommended Pym init code which violates .eslint standards
     const pym = new Pym.Child({polling: 100});
-    const path = /https?\:\/\/([^?]+)/.exec(pym.parentUrl);
-    this.props.getStream(path && path[1] || window.location);
+    const isURL = /https?\:\/\/([^?]+)/.test(pym.parentUrl);
+
+    this.props.getStream(isURL && pym.parentUrl || window.location);
```
Comment (impronunciable) [inline client/coral-embed-stream/src/CommentStream.js]: I'd add parenthesis just to clarify

File: bin/cli-serve (hunk 3 above)
Comment (wyattjoh) [reply bin/cli-serve:51]: Well, we want it to throw the error either way, that way the application will close.

**Pull request merged**